packages/angular-material/src/controls/range.renderer.ts

```diff
@@ -33,10 +33,11 @@ import { isRangeControl, RankedTester, rankWith } from '@jsonforms/core';
 @Component({
   selector: 'RangeControlRenderer',
   template: `
-    <div fxFlex fxLayout="column" [fxHide]="hidden">
+    <div [ngStyle]="{ display: hidden ? 'none' : '' }" class="range-control">
       <label class="mat-caption" style="color:rgba(0,0,0,.54)">{{
         label
       }}</label>
+      <!-- TODO: The 'tickInterval' property no longer exists -->
```

Comment:
Is there a problem with the current solution (using `showTickMarks`). If it currently works fine, I don't think we need this TODO

Reply:
Indeed, makes sense.